packages/card-service/src/tests/cardPayment.ts

```diff
@@ -0,0 +1,29 @@
+import { v4 as uuid } from 'uuid'
+import { IocContract } from '@adonisjs/fold'
+import { AppServices } from '../app'
+import { CreateCardPaymentOptions } from '../card-payment/service'
+import { CardPayment } from '../card-payment/model'
+
+export const randomCardPayment = (): CreateCardPaymentOptions => ({
+  requestId: uuid(),
+  requestedAt: new Date(),
+  cardWalletAddress: `https://wallet-${uuid().slice(0, 8)}.example/.well-known/pay`,
+  incomingPaymentUrl: `https://backend-${uuid().slice(0, 8)}.example/incoming-payments/${uuid()}`,
```

Comment:
_VERY_ nitpicky: with multi-tenancy OP payments urls now have a uuid in between the domain and the rest of the path, so just inserting a uuid to represent a tenant id in between the domain and the `/incoming-payments` part of the path would bring this more in line with a "real" example.